Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
+      activemodel (= 6.1.3)
+      activesupport (= 6.1.3)
+    activesupport (6.1.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-      zeitwerk (~> 2.2, >= 2.2.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
     builder (3.2.4)
-    concurrent-ruby (1.1.7)
+    concurrent-ruby (1.1.8)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.9.0)
-    i18n (1.8.5)
+    erubi (1.10.0)
+    i18n (1.8.9)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.8.5 → 1.8.9

[_[change-log](https://github.com/ruby-i18n/i18n/releases), [source-code](https://github.com/ruby-i18n/i18n)_]





<details>
<summary>Commits</summary>

A change of **43** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.8.5...v1.8.9).

These are the first 10 commits:
- (f6c610d) [The repository has moved to ruby-i18n organization](https://github.com/ruby-i18n/i18n/commit/f6c610d2fa53ec1f6df2a326cbbb3765994cba28)
- (d024044) [store fallbacks in Thread.current](https://github.com/ruby-i18n/i18n/commit/d0240440bf6616240523a215cd228a1d17740adb)
- (4709391) [Merge pull request #542 from jonian/threadsafe_fallbacks](https://github.com/ruby-i18n/i18n/commit/4709391dceab9096d5988576f93935843023a6ef)
- (91f1df2) [Merge pull request #539 from amatsuda/repo](https://github.com/ruby-i18n/i18n/commit/91f1df2339468791a74a875f76be0fbe3115fcbe)
- (dc8fc53) [Add support for Rails 6.1](https://github.com/ruby-i18n/i18n/commit/dc8fc53cd28928cf8f84b5cdf36162f4045f26c6)
- (ec56bf3) [Remove checks for Rails 5.0 and Rails 5.1](https://github.com/ruby-i18n/i18n/commit/ec56bf35fb8171e0351261a723d4be1ef452238a)
- (1fb723b) [Add Ruby 3.0.0 checks](https://github.com/ruby-i18n/i18n/commit/1fb723b87b3569b846f0b8d89134b98b5121222b)
- (ebca9c8) [Add gemfile for Rails 6.1](https://github.com/ruby-i18n/i18n/commit/ebca9c8e835492b4f39d06dd84eea0e70b53ffb6)
- (79a91e4) [Ruby 3.x and Rails 5.2.x are not incompatible](https://github.com/ruby-i18n/i18n/commit/79a91e41e90ebcc9bf3fe92fda3b1a3c09a8fa9e)
- (851009b) [Bump minitest versions](https://github.com/ruby-i18n/i18n/commit/851009b5f9de819ce38006b372ba9c68038900f8)


</details>